src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -96,23 +131,32 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime советуют использовать при записи даты в базу данных.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, TIME_ZONE_UKRAINE);
     }
 
     /**
      * Дана строка в виде "yyyymmdd".
      * Необходимо вернуть Optional даты в LocalDate формате
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        try {
+            return Optional.ofNullable(LocalDate.parse(date, FORMATTER));
```

Comment:
```suggestion
            return Optional.of(LocalDate.parse(date, ??? ));
```

there is already a predefined formatter that suits here

Reply:
done